packages/pocket-ic/src/lib.rs

```diff
@@ -1382,9 +1383,11 @@ To download the binary, please visit https://github.com/dfinity/pocketic."
     ));
     #[cfg(not(windows))]
     cmd.arg(port_file_path.clone());
-    if std::env::var("POCKET_IC_MUTE_SERVER").is_ok() {
-        cmd.stdout(std::process::Stdio::null());
-        cmd.stderr(std::process::Stdio::null());
+    if let Ok(mute_server) = std::env::var("POCKET_IC_MUTE_SERVER") {
+        if mute_server == "1" {
```

Comment:
How about checking for non-empty string instead?

Reply:
done